manim/utils/color.py

```diff
@@ -437,14 +437,17 @@ def average_color(*colors: Color) -> Color:
     return rgb_to_color(mean_rgb)
 
 
-def random_bright_color() -> Color:
-    color = random_color()
+def random_bright_color(random_seed=None) -> Color:
+    color = random_color(random_seed)
     curr_rgb = color_to_rgb(color)
     new_rgb = interpolate(curr_rgb, np.ones(len(curr_rgb)), 0.5)
     return Color(rgb=new_rgb)
 
 
-def random_color() -> Color:
+def random_color(random_seed=None) -> Color:
+    if random_seed != None:
+        range = random.Random(random_seed)
+        return list(Colors)[range.randint(0, len(list(Colors)))]
```

Comment:
Is there a particular reason for using a completely different implementation than in the `random_seed=None` case?

Reply:
No. I just wasn't sure how to implement the random seed logic